lib/active_model/serializer.rb

```diff
@@ -59,17 +59,11 @@ class << self
     end
 
     # @api private
-    def self.serializer_lookup_chain_for(klass)
-      chain = []
-
-      resource_class_name = klass.name.demodulize
-      resource_namespace = klass.name.deconstantize
-      serializer_class_name = "#{resource_class_name}Serializer"
-
-      chain.push("#{name}::#{serializer_class_name}") if self != ActiveModel::Serializer
-      chain.push("#{resource_namespace}::#{serializer_class_name}")
-
-      chain
+    def self.serializer_lookup_chain_for(klass, namespace = nil)
+      lookups = ActiveModelSerializers.config.serializer_lookup_chain
```

Comment:
I think it would be better for the lookup_chain to be a class_attribute or some such on the serializer

```ruby
class_attribute(:lookup_chain) {  ActiveModelSerializers.config.serializer_lookup_chain }
```